tests/e2e-kubernetes/README.md

```diff
@@ -0,0 +1,23 @@
+# Kubernetes tests
+
+This directory imports tests from kubernetes/kubernetes and enables:
+
+* External CSI tests, https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/external
+* CSI migration tests
+
+# Prerequisites
+- kubernetes 1.16+ AWS cluster
+
+# Run
+* External CSI tests: `FOCUS=External.Storage`.
+* CSI migration tests: `FOCUS=ebs-csi-migration`.
+
+```sh
+go test -v -timeout 0 ./... -kubeconfig=$HOME/.kube/config -report-dir=$ARTIFACTS -ginkgo.focus="$FOCUS" -ginkgo.skip="\[Disruptive\]" -gce-zone=us-west-2a
+```
+
+# Update dependencies
+```sh
+go mod edit -require=k8s.io/kubernetes@v1.15.3
```

Comment:
Which k/k version supports external tests? Update this accordingly?

Reply:
The version of installed kubernetes and tests should be the same. Running newer version of the tests against older kubernetes will test features that are not enabled by default or run regression tests for bugs that were not fixed yet. That's why 1.15.

As I wrote above, we need 1.17 cluster or even 1.18 to run some real tests. But I don't know how to install it with k8s-e2e-tester.